.github/workflows/test.yml

```diff
@@ -1,6 +1,8 @@
 name: Tests
 
-on: push
+on:
+  push:
+  workflow_dispatch:
```

Comment:
Does this actually change any behaviour in our repo?

Reply:
It adds ability to run a workflow on-demand, but it doesn't change/remove any of the behavior of the current triggers.
It looks like this in my fork
<img width="779" alt="image" src="https://github.com/user-attachments/assets/8324fc71-9cac-45dc-8ef7-ddc4ab03d595">